Modules/_pickle.c

```diff
@@ -2142,47 +2142,70 @@ save_float(PicklerObject *self, PyObject *obj)
     return 0;
 }
 
-/* No-copy code-path to write large contiguous data directly into the
-   underlying file object, bypassing the output_buffer of the Pickler. */
+/* Write large contiguous data directly into the underlying file object,
```

Comment:
Can you rephrase the comments now that this is called also when there is no underlying file object?
See comments in `pickle.py` for inspiration.